app/experimenter/experiments/tests/api/v5/test_mutations.py

```diff
@@ -16,143 +16,27 @@
 mutation($input: ExperimentInput!) {
     createExperiment(input: $input) {
         nimbusExperiment {
-            name
-            hypothesis
-            application
+            slug
         }
         message
-        status
     }
 }
 """
 
 
-UPDATE_EXPERIMENT_OVERVIEW_MUTATION = """\
+UPDATE_EXPERIMENT_MUTATION = """\
 mutation($input: ExperimentInput!) {
     updateExperiment(input: $input) {
-        nimbusExperiment {
```

Comment:
Don't we use the `nimbusExperiment` returned in mutation response to update the local cache with experiment changes on the client side?

Reply:
Didn't seem to in my local testing, can you link to the code that uses it?

Reply:
FXA does this, but IIRC we decided early on that we wanted the freshest experiment data and opted not to use the client cache at all. I'd be down for revisiting that, though.

Reply:
Yeah, [no cache](https://github.com/mozilla/experimenter/blob/main/app/experimenter/nimbus-ui/src/hooks/useExperiment.tsx#L49:L49)

Reply:
Yeah looks to me like it just refetches the whole experiment by slug on each page load and after each save:

<img width="1522" alt="Screen Shot 2021-02-16 at 1 44 02 PM" src="https://user-images.githubusercontent.com/119884/108107323-14e8c280-705d-11eb-9ebf-1bc20161eb21.png">

Reply:
This is mainly what my fuzzy recollection was pointing me to: https://www.apollographql.com/docs/react/data/mutations/#updating-a-single-existing-entity

But if we've got that disabled, then I guess yeah we're not doing client-side caching like that

Reply:
FWIW I would love to do use that and cut back on requests. I can't remember what the exact reason was for not using any cache, maybe just because something could change it externally, but I doubt it would cause problems.

Reply:
Yeah, we opted not to cache anything at all and to always refresh it per page in order to make sure the freshest experiment data was reflected. Not the most ideal, but I wrote a sorta mini-ADR in #3471 which is related, though I haven't read through this since then.